emr-serverless-sample/README.md

```diff
@@ -0,0 +1,112 @@
+# Running EMR Serverless Jobs with Java
+
+We will run a Java Spark job on EMR Serverless using a simple Java "Hello World" example in this example.
+
+## Prerequisites
+
+* LocalStack
+* `aws` CLI & `awslocal` script
+* Docker
+* Java and Maven
+
+## Installation
+
+Before creating the EMR Serverless job, we need to create a JAR file containing the Java code. To do this, run the following command:
+
+```bash
+cd hello-world
+mvn package
+```
+
+Next, we need to create an S3 bucket to store the JAR file. To do this, run the following command:
+
+```bash
+export S3_BUCKET=test
+awslocal s3 mb s3://$S3_BUCKET
+```
+
+You can now copy the JAR file to the S3 bucket:
+
+```bash
+awslocal s3 cp target/java-demo-1.0.jar s3://${S3_BUCKET}/code/java-spark/ 
+```
+
+## Creating the EMR Serverless Job
+
+Specify the ARN for the EMR Serverless job with the following command:
+
+```bash
+export JOB_ROLE_ARN=arn:aws:iam::000000000000:role/emr-serverless-job-role
+```
+
+We can now create an EMR Serverless application, which will run Spark 3.3.0. Run the following command:
+
+```bash
+awslocal emr-serverless create-application \
+  --type SPARK \
+  --name serverless-java-demo \
+  --release-label "emr-6.9.0" \
+    --initial-capacity '{
+        "DRIVER": {
+            "workerCount": 1,
+            "workerConfiguration": {
+                "cpu": "4vCPU",
+                "memory": "16GB"
+            }
+        },
+        "EXECUTOR": {
+            "workerCount": 3,
+            "workerConfiguration": {
+                "cpu": "4vCPU",
+                "memory": "16GB"
+            }
+        }
+    }'
+```
+
+You can retrieve the Application ID from the output of the command, and export it as an environment variable:
+
+```bash
+export APPLICATION_ID=<application-id>
```

Comment:
I would always escape `<` and `>` in code snippets, to avoid pipe shell redirects (even though it should be clear that this is a placeholder, some folks may be blindly copy/pasting the commands :) )
```suggestion
export APPLICATION_ID='<application-id>'
```